Osirium/trusty
# Move to Xenial 64 and push to artifactory

To test this:
- Delete contents of `debian-local` artifactory repo to ensure a clean state (It's not being used yet).
- `make debs && ARTIFACTORY_USR=[user] ARTIFACTORY_PSW=[password] make upload`. This should be idempotent.
- Check the contents of `debian-local` in artifactory. 

PS: The index is automatically made by artifactory, that's why it has been removed here.
